src/Stryker.Core/Stryker.Core/Reporters/TestStatisticsReporter/TestStatisticsReporter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using Stryker.Core.Mutants;
+using Stryker.Core.Options;
+using Stryker.Core.ProjectComponents;
+using Stryker.Core.Reporters.Json;
+using Stryker.Core.Testing;
+
+namespace Stryker.Core.Reporters
+{
+    public class TestStatisticsReporter: IReporter
```

Comment:
Is this reporter registered as a reporter and added to the list of reporter options?

Reply:
At this state, it is experimental. I used it to assess the interest of batching mutants. It is a useful tool that can lead to many interesting findings. But not for now. 
It should not have been part of this PR.
Removed